CONTRIBUTING.md

```diff
@@ -125,4 +125,4 @@ We have a set of [coding conventions](https://wiki.hyperledger.org/display/BESU/
 [private@pegasys.tech]: mailto:private@pegasys.tech
 [Gitter]: https://gitter.im/PegaSysEng/teku
 [CLA.md]: /CLA.md
-[Code Reviews]: /docs/code-reviews.md
+[Code Reviews]: /code-reviews.md
```

Comment:
I'd suggest we just delete this file - we don't actually follow what it says.  The PR policy is setup as GitHub config so you just have to do what it demands now anyway. :)